services/data/postgres_async_db.py

```diff
@@ -16,12 +16,14 @@
 from services.utils import DBConfiguration
 
 AIOPG_ECHO = os.environ.get("AIOPG_ECHO", 0) == "1"
-FEATURE_DB_TRIGGER_DISABLE = os.environ.get("FEATURE_DB_TRIGGER_DISABLE", 0) == "1"
-FEATURE_JOIN_DISABLE = os.environ.get("FEATURE_JOIN_DISABLE", 0) == "1"
 
 WAIT_TIME = 10
 OLD_RUN_FAILURE_CUTOFF_TIME = 60 * 60 * 24 * 1000 * 14  # 2 weeks (in milliseconds)
 
+# Create database triggers automatically, disabled by default
+# Enable with env variable `DB_TRIGGER_CREATE=1`
+DB_TRIGGER_CREATE = os.environ.get("DB_TRIGGER_CREATE", 0) == "1"
```

Comment:
default still remains 'false' so metadata service needs to specifically have the env var set so it creates triggers when running on a write master? Not an issue, just a configuration hurdle that should probably be noted.